src/MeshKernelNET/Api/IMeshKernelApi.cs

```diff
@@ -1631,8 +1631,26 @@ public interface IMeshKernelApi : IDisposable
         /// <param name="secondIndex">The index of the second vertex</param>
         /// <param name="disposableGeometryListOut">The refined polygon</param>
         /// <returns>Error code</returns>
-        int PolygonLinearRefine(int meshKernelId, in DisposableGeometryList disposableGeometryListIn, int firstIndex,
-                          int secondIndex, ref DisposableGeometryList disposableGeometryListOut);
+        int PolygonLinearRefine(int meshKernelId, 
+                                in DisposableGeometryList disposableGeometryListIn, 
+                                int firstIndex,
+                                int secondIndex, 
+                                ref DisposableGeometryList disposableGeometryListOut);
+
+        /// <summary>
+        /// Snaps part of a polygon to a land boundary
+        /// </summary>
+        /// <param name="meshKernelId">Id of the mesh state</param
+        /// <param name="landboundaries">The land boundaries</param>

```

Comment:
Does this work with multiple land boundaries? If not please change to singular also in implementation